blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/horizontal-links/_children/link.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+function getLocation(uri) {
+  let url;
+  if (typeof window === 'undefined') {
+    url = new URL(uri, 'http://example.com');
```

Comment:
not sure I totally understand `example.com` included here as the base? for testing. for ssr?

Reply:
yes - this was included in the copyover from the links bar block and is used for testing.

Reply:
can you remove it if it's just used for testing? sorry, it just seems unnecessary to generate this for all ssr renders

Reply:
the URL object need a base domain for work. the URL is used to parse the url on `fixTrailingSlash` function to do some checks. The final url is never used.

Reply:
ok if this is absolutely necessary